packages/web-app-files/src/components/Spaces/QuotaModal.vue

```diff
@@ -98,22 +111,17 @@ export default {
         {
           displayValue: '1000',
           displayUnit: 'GB',
-          value: 10000 * Math.pow(10, 9)
+          value: 1000 * Math.pow(10, 9)
```

Comment:
Wow, this was so wrong

Reply:
Are we doing this correctly here? Just remember us using a `filesize` library in other parts of the project

Reply:
Filesize lib is for human-readable transformations, we don't want to have the magic converting 1000GB -> 1 TB
(see related issue).

We also have the requirement to use GB over GiB

Reply:
Aye, thanks for the clarification!